From 5944e9908fc12d69d19a1d24128cbc6d1a423c3d Mon Sep 17 00:00:00 2001
From: Xin Ouyang <Xin.Ouyang@windriver.com>
Date: Tue, 18 Jun 2013 12:29:00 +0800
Subject: [PATCH] policycoreutils: fix build strict-prototypes failure

| policy.c:90:6: error: function declaration isn't a prototype
[-Werror=strict-prototypes]
| cc1: all warnings being treated as errors
| error: command 'i586-poky-linux-gcc' failed with exit status 1
| make[1]: *** [python-build] Error 1

Upstream-Status: pending

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 sepolicy/policy.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sepolicy/policy.c b/sepolicy/policy.c
index 4eca22d..e454e75 100644
--- a/sepolicy/policy.c
+++ b/sepolicy/policy.c
@@ -87,7 +87,7 @@ static PyMethodDef methods[] = {
 	{NULL, NULL, 0, NULL}	/* sentinel */
 };
 
-void init_policy() {
+void init_policy(void) {
 PyObject *m;
 m = Py_InitModule("_policy", methods);
 init_info(m);
-- 
1.7.9.5

